Plugins~/Build/VsDevCmd_2022.bat

```diff
@@ -1,6 +1,7 @@
-for /f "usebackq tokens=*" %%i in (`"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe" /version [15.0^,16^) /property installationPath`) do (
+for /f "usebackq tokens=*" %%i in (`"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe" /version 17^ /property installationPath`) do (
```

Comment:
The version argument should be `[17.0^,18^)` , at least according to the help section, shouldn't it ?

```
  -version arg   A version range for instances to find. Example: [15.0,16.0) will find versions 15.*.
                 See https://aka.ms/vswhere/versions for more information about versions.
```

Reply:
Updated now